projects/pgai/pgai/vectorizer/embedders/voyageai.py

```diff
@@ -38,8 +38,8 @@ def voyage_token_counter(model: str) -> Callable[[str], int] | None:
     try:
         tokenizer: Tokenizer = client.tokenizer(model)
         return lambda text: len(tokenizer.encode(text).tokens)
-    except BaseException:
-        logger.warn(f"Tokenizer for model '{model}' not found")
+    except Exception as e:
+        logger.warn(f"Tokenizer for model '{model}' not found: {e}")
```

Comment:
I think generally speaking simply using `exc_info=True` is going to look nicer than printing the stringified version. It also shows the full stack and chain of exceptions if there was one.